scripts/third_party_size_check.py

```diff
@@ -21,35 +21,62 @@
 
 import os
 import sys
+import yaml
 
 THIRD_PARTY_PATH = os.path.join(os.getcwd(), 'third_party')
 THIRD_PARTY_SIZE_LIMIT = 7000
 
 
-def _check_size_in_dir(dir_path):
+def _get_skip_files_list():
+    """This function returns the list of the files which are skipped when
+    Oppia is deployed to GAE.
+
+    Returns:
+        list. The list of skip_files directories.
+    """
+    with open('./app.yaml', 'r') as app_yaml:
+        try:
+            app_yaml_dict = yaml.safe_load(app_yaml)
+        except yaml.YAMLError as exc:
```

Comment:
Please change it to `except yaml.YAMLError as yaml_exception:`

Reply:
Done Thanks!